src/main/java/com/olgaboiar/mint/Server.java

```diff
@@ -67,29 +45,13 @@ private Request parseRequest(List<String> clientInput) throws IOException {
     }
 
     private Response prepareResponse(Request parsedRequest) throws IOException {
-        Response response = new Router().route(parsedRequest);
+        RoutesConfiguration serverRoutes = new RoutesConfiguration();
```

Comment:
The variable name doesn't match the class name.

Reply:
fixed in https://github.com/olgaboiar/MintHTTPServer/pull/10/commits/6f1b1c6f35fd26884d6ac08854553aa2fa27f1c3